box-content-sdk/src/main/java/com/box/androidsdk/content/BoxApiFile.java

```diff
@@ -541,4 +555,89 @@ public BoxApiFile(BoxSession session) {
     public BoxRequestsFile.FilePreviewed getFilePreviewedRequest(String fileId) {
         return new BoxRequestsFile.FilePreviewed(fileId, getPreviewFileUrl(), mSession);
     }
+
+    /**
+     * Gets a request that creates an upload session for uploading a new file or a new file version
```

Comment:
According to this comment, this API can be used for new file or new versions.
According to the API documentation, the URL for new versions ("files/<fileid>/upload_sessions)" is not the same as new files ("files/upload_sessions"). Please check as we might need some other API for uploading new file's version.

Reply:
Think they updated this..I'll add it.

Reply:
Done